From d46cd5d06976e409efd18f7bc215bfbc907c0ed5 Mon Sep 17 00:00:00 2001
From: Trilok Soni <tsoni@codeaurora.org>
Date: Thu, 8 Nov 2012 14:49:24 +0530
Subject: [PATCH 36/75] msm: board-qrd7627a: Regulator configuration for 8625q device

In 8625Q the acpuclk voltage comes from regulator. This patch adds
the regulator support for the 8625Q device.

Change-Id: Id1b8aa06feb5ce56b6207641321db85a10b5a8d7
Signed-off-by: Trilok Soni <tsoni@codeaurora.org>
Signed-off-by: Utsab Bose <ubose@codeaurora.org>
Signed-off-by: Tirupathi Reddy <tirupath@codeaurora.org>
---
 arch/arm/mach-msm/board-qrd7627a.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-msm/board-qrd7627a.c b/arch/arm/mach-msm/board-qrd7627a.c
index 1cfebf4..8ef3f94 100644
--- a/arch/arm/mach-msm/board-qrd7627a.c
+++ b/arch/arm/mach-msm/board-qrd7627a.c
@@ -592,6 +592,10 @@ static struct platform_device qrd_vreg_gpio_ext_1p8v __devinitdata = {
 /* Regulator configuration for the NCP6335D buck */
 struct regulator_consumer_supply ncp6335d_consumer_supplies[] = {
 	REGULATOR_SUPPLY("ncp6335d", NULL),
+	/* TO DO: NULL entry needs to be fixed once
+	 * we fix the cross-dependencies.
+	*/
+	REGULATOR_SUPPLY("vddx_cx", NULL),
 };
 
 static struct regulator_init_data ncp6335d_init_data = {
-- 
1.7.0.4

